pkg/provider/cloud/kubevirt/network_policy.go

```diff
@@ -23,12 +23,13 @@ import (
 	kubermaticv1 "k8c.io/kubermatic/v2/pkg/apis/kubermatic/v1"
 	"k8c.io/reconciler/pkg/reconciling"
 
+	corev1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func clusterIsolationNetworkPolicyReconciler() reconciling.NamedNetworkPolicyReconcilerFactory {
+func clusterIsolationNetworkPolicyReconciler(isCustomImagesOrCloningEnabled bool) reconciling.NamedNetworkPolicyReconcilerFactory {
```

Comment:
It would be better to create a new network policy instead of extending the cluster-isolation.
Could we do that? It will be easier to manage. It should be called `cluster-images`